### ISSUE TITLE
Scoop Up put 2 creatures into different archives

### ISSUE THREAD

**Issue by abeautifuljune** [keyteki/keyteki#3792]
My opponent played Talent Scout, giving me control of it. I played Scoop Up targeting their Talent Scout under my control and their Shrewd Investor. I would expect both cards to be put into my archive, however the Talent Scout was archived by my opponent and the Shrewd Investor was archived by me. Neither of us had any archives prior to this.

**Comment by strib** [comment]
This is not a bug.  Scoop Up (unlike something like Zookeeper) doesn't allow for the opponent's cards to go into your archives, because there is no text putting it back into your opponent's hand if it leaves your archives (therefore it would go to your hand, which is illegal).  So instead it goes to its owner's appropriate play zone, which is their archives.  This is discussed in the commentary on the Archon Arcana page: https://archonarcana.com/Scoop_Up

**Comment by abeautifuljune** [comment]
It DID allow my opponent's Shrewd Investor to enter my archives.

**Comment by strib** [comment]
Yes that is correct, because Scoop Up has the "leaves archives" condition for the enemy creature. It doesn't have it for the friendly creature.

**Comment by dmikalova** [comment]
This has been fixed for the errata to work as expected.
